page-container/index-page/IndexPageContainer.tsx

```diff
@@ -0,0 +1,108 @@
+import { Box, Grid, makeStyles } from '@material-ui/core';
+import { Header, Footer } from 'pedialab-shared/components';
+import {
+  HeroContent,
+  OurCompany,
+  SoftwareDelivery,
+  VendorDifference,
+  TechleadaaS,
+  ProjectScopes,
+  TeamUpWithTechlead,
+  DeveloperSupport,
+  OurTeam,
+  CompaniesWeWorkWith,
+  WhereWeServe,
+  CaseStudies,
+  ProjectPhrases,
+  Testimonial,
+  Contact
+} from 'pedialab-pages/index-page';
+
+const useStyle = makeStyles({
+  heroContent: {
+    marginTop: '7rem'
+  },
+  ourCompany: {
+    marginTop: '7.5rem',
+    paddingLeft: '8.8%'
+  },
+  softwareDelivery: {
+    marginTop: '14rem'
+  },
+  vendorDifference: {
+    marginTop: '13rem'
+  },
+  techleadaaS: {
+    marginTop: '14.65rem'
+  },
+  projectScopes: {
+    marginTop: '15rem'
+  },
+  teamUpWithTechlead: {
+    marginTop: '13.75rem'
+  },
+  developerSupport: {
+    marginTop: '15.625rem'
+  },
+  ourTeam: {
+    marginTop: '15.75rem'
+  },
+  companiesWeWorkWith: {
+    marginTop: '20rem'
+  },
+  whereWeServe: {
+    marginTop: '16.4375rem'
+  },
+  caseStudies: { marginTop: '24.3125rem' },
+  projectPhrases: {
+    marginTop: '19.875rem'
+  },
+  testimonial: {
+    marginTop: '12.75rem'
+  },
+  contact: { marginTop: '13rem' },
+  footer: {
+    marginTop: '9.4rem'
+  }
+});
+
+const IndexPageContainer = () => {
+  const classes = useStyle();
+  return (
+    <>
+      <Box mx="10%">
+        <Grid container direction="column">
+          <Header />
+          <HeroContent className={classes.heroContent} />
+          <OurCompany className={classes.ourCompany} />
+          <SoftwareDelivery className={classes.softwareDelivery} />
+          <VendorDifference className={classes.vendorDifference} />
+        </Grid>
+      </Box>
+      <Box ml="10%">
+        <TechleadaaS className={classes.techleadaaS} />
+      </Box>
+      <Box mx="10%">
```

Comment:
IMO <Container /> is probably a better fit if you want to set responsive margin 
https://material-ui.com/components/container/

Reply:
Yes, I'll try to replace from `<Box mx>`  to `<Container>`